src/wren_compiler.c

```diff
@@ -789,6 +789,7 @@ static void nextToken(Parser* parser)
         makeToken(parser, TOKEN_LINE);
         return;
 
+      case '\t':
       case ' ':
         // Skip forward until we run out of whitespace.
         while (peekChar(parser) == ' ') nextChar(parser);
```

Comment:
I think this should be:

    while (peekChar(parser) == ' ' || peekChar(parser) == '\t') nextChar(parser);